apps/mobile-alphabet/android/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
When Ruby, Java, JavaScript, TypeScript, Groovy, and Objective-C can't get the job done, bring in the big guns. Hello BASH!